artifactory/utils/utils.go

```diff
@@ -159,6 +160,27 @@ func CreateDistributionServiceManager(artDetails *config.ServerDetails, isDryRun
 	return distribution.New(serviceConfig)
 }
 
+func CreateAccessServiceManager(artDetails *config.ServerDetails, isDryRun bool) (*access.AccessServicesManager, error) {
```

Comment:
artDetails --> serviceDetails
("art" stands for Artifactory, and we're creatiung a service manager for Access, and not Artifactory).
Let's make the same change in the ```CreateDistributionServiceManager``` function in this file.